lib/players/computer.rb

```diff
@@ -0,0 +1,11 @@
+module Players
+  class Computer < Player
+
+    attr_accessor :board
+
+    def move(board)
+      "1"
+    end
```

Comment:
Doesn't seem like the computer can do much here... how could we at least build some logic so the computer can play the game -- even if it makes bad choices for moves?